kubectl-plugin/pkg/cmd/get/get_cluster.go

```diff
@@ -135,6 +137,11 @@ func printClusters(rayclusterList *rayv1.RayClusterList, output io.Writer) error
 		if raycluster.GetCreationTimestamp().Time.IsZero() {
 			age = "<unknown>"
 		}
+		lastConditionType := ""
+		lastCondition := util.LastCondition(raycluster.Status.Conditions)
```

Comment:
I show the most recent Condition. Lmk if this is what you want.

Reply:
Or if logic should be show the first of these that is present regardless of the Condition's other fields as @kevin85421 [described here](https://ray.slack.com/archives/C02GFQ82JPM/p1738690789583719?thread_ts=1738681240.563379&cid=C02GFQ82JPM). RayClusterProvisioned > HeadPodReady > RayClusterReplicaFailure > Suspended > Suspending